std/examples/ws/server.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
```

Comment:
See https://github.com/denoland/deno/pull/5087#discussion_r422486964.

Reply:
Yeah I know about that but I think it's strange as well. Actually we have few examples across std modules now. If we'll add example more by that policy, we have many `example_` file in whole directories.

Reply:
I think `std/examples` is just to showcase `deno`/`std`'s capabilites for beginners. I actually agree that an example to teach a more technical API, for people wanting to learn that API specifically, should be found in its respective directory.

Reply:
I think it makes more sense for each module directory to have its own examples. Otherwise we're going to have the problem of mirroring the std tree in the examples directory, requiring users to browse both at the same time. std/examples is just a couple of small high level examples like welcome.ts that don't belong elsewhere.

Reply:
OK, I'll revert the changes.